### PR TITLE
remove cheerio dependency

### PR DESCRIPTION
**Pull request by mallendeo**

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -223,3 +224,18 @@ exports.parseFormats = function(info, debug) {
...
+
+/**
+ * Get video description from html
+ *
+ * @param {String} html
+ * @return {String}
+ */
+ exports.getVideoDescription = function(html) {
+   var regexString = '<p.*?id="eow-description".*?>(.+?)<\/p>'
+                   + '[\\n\\r\\s]*?<\/div>';
```
Comment (fent) [inline lib/util.js]: why not make this a compile time regex object? rather than using the dynamic RegExp class.

File: lib/util.js [hunk 2]
```diff
@@ -223,3 +224,18 @@ exports.parseFormats = function(info, debug) {
...
+   var regex = new RegExp(regexString, 'igm');
```
Comment (fent) [inline lib/util.js]: I don't think the "g" option is needed here since it's only one match

File: lib/util.js [hunk 3]
```diff
@@ -223,3 +224,18 @@ exports.parseFormats = function(info, debug) {
...
+   var description = regex.exec(html.match(regex));
```
Comment (fent) [inline lib/util.js]: Using `RegExp#exec()` on the `String#match()` result??

File: package.json [hunk 4]
```diff
@@ -17,7 +17,7 @@
     "test": "mocha -R spec -t 16000 test/*-test.js"
   },
   "dependencies": {
-    "cheerio": "~0.19.0",
+    "entities": "^1.1.1",
```
Comment (fent) [inline package.json]: There's currently another PR (#34) that uses html-entities instead, what do you think?

Comment (mallendeo) [inline package.json]: Updated. It'll be better for merge, also take less size in the browserify bundle, so it's fine.

**Pull request merged**